### ISSUE TITLE
As a user, I should be able to effect my sound with delay

### ISSUE THREAD

**Issue by soiferj** [nspilman/web-synth#54]

**Comment by joshstruve** [comment]
Are there other, similar effects such as reverb that you would want to add? I'm thinking that this creates a new "Effects" category in addition to "Waveform", "Envelope", "Oscillator" and "Filter". Otherwise, the new effects don't cleanly fit into any of the existing categories. Please let me know your thoughts.

**Comment by soiferj** [comment]
Yes an effects section would be great! We can move the current distortion there too.

**Comment by joshstruve** [comment]
Confirmed, thanks.
